Sources/CodeEditTextView/TextView/TextView.swift

```diff
@@ -314,12 +314,14 @@ public class TextView: NSView, NSTextContent {
     open override func becomeFirstResponder() -> Bool {
         isFirstResponder = true
         selectionManager.cursorTimer.resetTimer()
+        needsDisplay = true
         return super.becomeFirstResponder()
     }
 
     open override func resignFirstResponder() -> Bool {
         isFirstResponder = false
         selectionManager.removeCursors()
+        needsDisplay = true
```

Comment:
What is the `needsDisplay` variable doing?

Reply:
Just wondering because its being set to true on both `becomeFirstResponder` and `resignFirstResponder` and its not referenced anywhere else in the PR.

Reply:
`needsDisplay` tells MacOS to redraw the view which is needed to redraw the selections in a new color.